PY_Homework007/task34.py

```diff
@@ -0,0 +1,40 @@
+# Задача 34: Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает,
+# Вам стоит написать программу.
+# Винни-Пух считает,что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое.
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами.
+# Фразы отделяются друг от друга пробелами.
+# Стихотворение Винни-Пух вбивает в программу с клавиатуры.
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке.
+
+# Пример№1:
+# Ввод: пара-ра-рам рам-пам-папам па-ра-па-дам
+# Вывод: парам пам-пам
+
+# Пример№2:
+# Ввод: пара-ра-рам рам-пам-папам па-ра-па
+# Вывод: пам парам
+
+def check_rhythm(poem):
+    # Разделяем стихотворение на отдельные слова
+    words = poem.split()
+
+    # Функция для подсчета числа гласных букв в слове
+    count_vowels = lambda word: sum(1 for letter in word if letter.lower() in 'аеёиоуыэюя')
```

Comment:
Чуть поправлю
```suggestion
    words = poem.lower().split()

    # Функция для подсчета числа гласных букв в слове
    count_vowels = lambda word: sum(letter in 'аеёиоуыэюя' for letter in word)
```
На мой взгляд так лучше. Подумайте что я сделал с 23 строкой